tests/functional/cylc-config/00-simple/section2.stdout

```diff
@@ -15,10 +15,14 @@
     execution time limit = 
     submission polling intervals = 
     submission retry delays = 
+    run mode = 
```

Comment:
I may be getting my wires crossed (please correct me if I am), but I think the idea was for `'live'` to be the default `run mode`?